padle/view/globalstats_view.py

```diff
@@ -0,0 +1,118 @@
+
+    def serialize(self):
+        ret = super().serialize()
+        ret.update({
+            'current_day': self.current_day,
+            'reaction_list': self.reaction_list,
+            'num_days': self.num_days,
+            'query_settings': self.query_settings.serialize()
+        })
+        return ret
+
+    @staticmethod
+    def increment_page(ims):
+        if ims['current_day'] < ims['num_days']:
+            ims['current_day'] = ims['current_day'] + 1
+        else:
+            ims['current_day'] = 1
+
+    @staticmethod
+    def decrement_page(ims):
+        if ims['current_day'] > 1:
+            ims['current_day'] = ims['current_day'] - 1
+        else:
+            ims['current_day'] = ims['num_days']
+
+    @classmethod
+    async def deserialize(cls, dbcog, _user_config: UserConfig, padle_cog, ims: dict):
+        original_author_id = ims['original_author_id']
+        menu_type = ims['menu_type']
+        reaction_list = ims['reaction_list']
+        current_day = ims['current_day']
+        num_days = ims['num_days']
+        query_settings = QuerySettings.deserialize(ims.get('query_settings'))
+        daily_scores_list = await padle_cog.config.save_daily_scores()
+        cur_day_scores = await padle_cog.config.all_scores()
+        if num_days == current_day:
+            return GlobalStatsViewState(original_author_id, menu_type, query_settings, "", current_day=current_day,
+                                        num_days=num_days, reaction_list=reaction_list, monster=None,
+                                        stats=cur_day_scores)
+        else:
+            info = daily_scores_list[current_day - 1]
+            m = dbcog.get_monster(int(info[0]))
+            return GlobalStatsViewState(original_author_id, menu_type, query_settings, "", current_day=current_day,
+                                        num_days=num_days, reaction_list=reaction_list, monster=m,
+                                        stats=info[1])
+
+
+class GlobalStatsView:
+    VIEW_TYPE = 'PADleGlobalStats'
+
+    @classmethod
+    def get_pages_footer(cls, state):
+        return "Day " + str(state.current_day) + "/" + str(state.num_days)
+    
+    @staticmethod
+    def embed(state: GlobalStatsViewState):
+        description = GlobalStatsView.get_description(state)
+        if state.num_days == state.current_day:
+            return EmbedView(
+                EmbedMain(
+                    title=f"**PADle #{state.current_day} Stats**",
+                    description=description,
+                    color=state.query_settings.embedcolor,
+                ),
+                embed_footer=embed_footer_with_state(state, text=GlobalStatsView.get_pages_footer(state)))
+        else:
+            return EmbedView(
+                EmbedMain(
+                    title=f"**PADle #{state.current_day} Stats**",
+                    description="**" + MonsterHeader.menu_title(state.monster).to_markdown() + "**\n" + description,
```

Comment:
theres a BoldText object in discordmenu that you can use

`Box(BoldText(that stuff), Box(description))` should work